### PR TITLE
Remove obsolete code blocks and support organization qualified URLs

### PR DESCRIPTION
**Pull request by sadilchamishka**
### Proposed changes in this pull request

$subject

### CODE REVIEW

File: components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/util/Utils.java [hunk 1]
```diff
@@ -88,20 +88,15 @@ public static boolean isUserBelongsToRequestedTenant(AuthenticationContext authe
                     Constants.AUTH_CONTEXT_OAUTH_APP_PROPERTY);
             tenantDomain = OAuth2Util.getTenantDomainOfOauthApp(oAuthAppDO);
         }
-        return tenantDomainFromURLMapping.equals(tenantDomain);
-    }
-
-    public static boolean isUserAuthorizedForOrganization(AuthenticationContext authenticationContext, Request request) {
```
Comment (sadilchamishka) [inline components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/util/Utils.java:94]: When reverting this PR again if stop supporting organization qualified URLs, only revert the changes done to this method.

**Review by AnuradhaSK — APPROVED** [review]
